docs/source/spelling_wordlist.txt

```diff
@@ -70,3 +70,18 @@ Pieter
 Abbeel
 et
 al
+params
+Init
+eval
+cfgs
+Richard S. Sutton
+David McAllester
+Satinder Singh
+Yishay Mansour
```

Comment:
One word per line.

Reply:
It has been modified according to your requirement.